src/utils/helpers.js

```diff
@@ -0,0 +1,16 @@
+import { isNil } from 'lodash';
```

Comment:
Calling it helpers inside utils is like `helpers/helpers`. What about `utils/numbers.js`?